homework-submission/HomeWork/main.js

```diff
@@ -0,0 +1,73 @@
+// create the Header component
+class Header extends React.Component {
+  render() {
+    return <h2 className="header">{this.props.title}</h2>;
+  }
+}
+
+// create the list items component
+class List extends React.Component {
```

Comment:
Clear list item component! Think another time of the name of the component though. The component is not a List in my opinion. Also could you describe the props this component receives? And for each prop what its purpose/effect is?

Reply:
Thanks @ArcoMul for your notes , i modify the code according your advice .